cmd/ocm/gcp/verify-wif-config.go

```diff
@@ -0,0 +1,49 @@
+package gcp
+
+import (
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// NewVerifyWorkloadIdentityConfiguration provides the "gcp verify wif-config" subcommand
+func NewVerifyWorkloadIdentityConfiguration() *cobra.Command {
+	verifyWorkloadIdentityCmd := &cobra.Command{
+		Use:   "wif-config [ID|Name]",
+		Short: "Verify a workload identity federation configuration (wif-config) object.",
+		RunE:  verifyWorkloadIdentityConfigurationCmd,
+	}
+
+	return verifyWorkloadIdentityCmd
+}
+
+func verifyWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) error {
+	key, err := wifKeyFromArgs(argv)
+	if err != nil {
+		return err
+	}
+
+	// Create the client for the OCM API:
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return errors.Wrapf(err, "Failed to create OCM connection")
+	}
+	defer connection.Close()
+
+	// Verify the WIF configuration exists
+	wif, err := findWifConfig(connection.ClustersMgmt().V1(), key)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get wif-config")
+	}
+
+	// Verify the WIF configuration is valid
+	response, err := connection.ClustersMgmt().V1().GCP().WifConfigs().WifConfig(wif.ID()).Status().Get().Send()
+	if err != nil {
+		return errors.Wrapf(err, "failed to verify wif-config")
+	}
+	if !response.Body().Configured() {
+		return errors.Wrapf(errors.New(response.Body().Description()), "failed to verify wif-config")
+	}
+
```

Comment:
when configured==true(backend returns 200) pls consider printing some txt to indicate verification succeeded

Reply:
I added the following text
```
➜  ocm-cli git:(OCM-12286) ✗ go run ./cmd/ocm gcp verify wif-config otto
WIF configuration is valid
```